fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1755,18 +1758,25 @@ void ReadYourWritesTransaction::atomicOp( const KeyRef& key, const ValueRef& ope
 }
 
 void ReadYourWritesTransaction::set( const KeyRef& key, const ValueRef& value ) {
-	if (key == LiteralStringRef("\xff\xff/reboot_worker")){
-		BinaryReader::fromStringRef<ClientWorkerInterface>(value, IncludeVersion()).reboot.send( RebootRequest() );
-		return;
-	}
-	if (key == LiteralStringRef("\xff\xff/reboot_and_check_worker")){
-		BinaryReader::fromStringRef<ClientWorkerInterface>(value, IncludeVersion()).reboot.send( RebootRequest(false, true) );
-		return;
-	}
+
 	if (key == metadataVersionKey) {
 		throw client_invalid_operation();
 	}
 
+	if (specialKeys.contains(key)) {
+		if (key == LiteralStringRef("\xff\xff/reboot_worker")){
+			BinaryReader::fromStringRef<ClientWorkerInterface>(value, IncludeVersion()).reboot.send( RebootRequest() );
```

Comment:
I think we should probably remove or update these special key writes for api version 700. `set` shouldn't have any immediate side effects in the new special key space

Reply:
We can discuss this in person.
Since the usage of it is against the semantics we have now, shall we just remove it in `700` and instead add a C API for it. 
Not sure if it is safe to deprecate it since it has been used for a long time.